backend/app/api/endpoints/admin.py

```
@@ -0,0 +1,427 @@
+        return {
+            "message": "Investor profile created successfully",
+            "investor_id": str(investor_profile.id),
+            "user_id": str(investor_user.id),
+            "created": True,
+        }
+
+    except Exception as e:
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"Failed to create investor: {str(e)}",
+        )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)